spec.html

```diff
@@ -23392,7 +23391,7 @@ <h1>
     <emu-clause id="sec-runtime-semantics-instantiateordinaryfunctionexpression" type="sdo">
       <h1>
         Runtime Semantics: InstantiateOrdinaryFunctionExpression (
-          optional _name_: unknown,
+          optional _name_: a String,
```

Comment:
This would also have to be updated if we expand the type of the `NamedEvaluation` parameter as I suggest above.

Reply:
Yup. If I'm not mistaken, this includes `ClassDefinitionEvaluation`, which might then pass a Private Name to `CreateBuiltinFunction`, which doesn't currently accept them. But I don't think there's a problem with adding it.